### ISSUE TITLE
CSGO Server is not working on Ubuntu 14.04

### ISSUE THREAD

**Issue by tuckdesign** [GameServerManagers/LinuxGSM#1347]
Trying to install csgo on in house computer for kids to play over lan. Installed csgo server on ubuntu 14.04 (through script) and when I start it with csgoserver.sh all seems fine (no errors and psa aux | grep csgo shows 2 tasks are running), but server is not there: 
netstat -atunp | grep srcds_linux returns nothing
netstat --listen doesn't show anything listening on csgo ports
https://hastebin.com/akumikiyic

**Comment by UltimateByte** [comment]
Wrong place to post + some more details are gonna be needed.
https://github.com/GameServerManagers/LinuxGSM/wiki/Support

**Comment by UltimateByte** [comment]
See also: https://github.com/GameServerManagers/LinuxGSM/wiki/Troubleshooting

**Comment by tuckdesign** [comment]
Where else can I submit a bug?
Here is what I get in log:
Server will auto-restart if there is a crash.
LD_LIBRARY_PATH=/home/krunoslav/Steam/serverfiles/bin:/home/krunoslav/Steam/serverfiles:/home/krunoslav/Steam/serverfiles/bin:
Illegal instruction (core dumped)
BFD: Warning: /home/krunoslav/Steam/serverfiles/core is truncated: expected core file size >= 1073152, found: 933888.
Python Exception <class 'gdb.MemoryError'> Cannot access memory at address 0xbfa5d37c: 
debug.cmds:1: Error in sourced command file:
Cannot access memory at address 0xbfa5d37c
email debug.log to linux@valvesoftware.com
Wed Mar  1 22:45:01 CET 2017: Server restart in 10 seconds
LD_LIBRARY_PATH=/home/krunoslav/Steam/serverfiles/bin:/home/krunoslav/Steam/serverfiles:/home/krunoslav/Steam/serverfiles/bin:
Illegal instruction (core dumped)
BFD: Warning: /home/krunoslav/Steam/serverfiles/core is truncated: expected core file size >= 1073152, found: 933888.
Python Exception <class 'gdb.MemoryError'> Cannot access memory at address 0xbf96ea8c: 
debug.cmds:1: Error in sourced command file:
Cannot access memory at address 0xbf96ea8c
email debug.log to linux@valvesoftware.com

and here are 'details':
Distro Details
========================================================================================================================================================================================================
Distro:    Ubuntu 14.04.5 LTS
Arch:      i686
Kernel:    3.13.0-110-generic
Hostname:  tuckdesign-ubuntu
tmux:      tmux 2.0
GLIBC:     2.19

Performance
========================================================================================================================================================================================================
Uptime:    0d, 11h, 49m
Avg Load:  0.00, 0.01, 0.05

Mem:       total   used   free     cached
Physical:  2.0G    1.8G   165M     1.5G
Swap:      2.0G    0B     2.0G

Storage
========================================================================================================================================================================================================
Filesystem:      /dev/sda1
Total:           90G
Used:            18G
Available:       68G
LinuxGSM Total:  13G
Serverfiles:     13G

Counter-Strike: Global Offensive Server Details
========================================================================================================================================================================================================
Server name:      LinuxGSM
Server IP:        192.168.1.200:27015
Server password:  NOT SET
RCON password:    admingRK4waFv
Maxplayers:       16
Game mode:        0
Tick rate:        64
Status:           OFFLINE

csgoserver.sh Script Details
========================================================================================================================================================================================================
Service name:           csgo-server
csgoserver.sh version:  170219
User:                   krunoslav
GLIBC required:         2.3.6
Email alert:            off
Pushbullet alert:       off
Update on start:        off
Location:               /home/krunoslav/Steam
Config file:            /home/krunoslav/Steam/serverfiles/csgo/cfg/csgo-server.cfg

Backups
========================================================================================================================================================================================================
No Backups created

Command-line Parameters
========================================================================================================================================================================================================
./srcds_run -game csgo -usercon -strictportbind -ip 192.168.1.200 -port 27015 +clientport 27005 +tv_port 27020 -tickrate 64 +map de_mirage +servercfgfile csgo-server.cfg -maxplayers_override 16 +mapgroup mg_active +game_mode 0 +game_type 0 +host_workshop_collection  +workshop_start_map  -authkey 

Ports
========================================================================================================================================================================================================
Change ports by editing the parameters in:
csgoserver.sh

Useful port diagnostic command:
netstat -atunp | grep srcds_linux

DESCRIPTION  DIRECTION  PORT   PROTOCOL
> Game/RCON  INBOUND    27015  tcp/udp
> SourceTV   INBOUND    27020  udp
< Client     OUTBOUND   27005  udp

Status:	OFFLINE

**Comment by UltimateByte** [comment]
> Where else can I submit a bug?
1) This isn't an LGSM bug, and GitHub is for LGSM Development and fixes only.
2) Maybe read the link(s) I provided before asking questions I already answered, unless your goal is just to piss me off by abusing of my time.

Your 11h, 49m uptime tells a lot about your abilities to manage a server.
At this point, what should matter to you is the logs output, or debug behavior. But obviously, you're not making any effort and didn't bother ready anything from the links I provided to you. And since you cannot read, you don't know how to 1) Help yourself 2) If you can't help yourself, how to help us helping you.

To sum up with one word: READ.

**Comment by tuckdesign** [comment]
Sorry if you are pissed of, just asked for help. Won't bother you any more. Thanks for not helping. 
BTW, I've read links you posted and they don't offer solution (tried), did debug and posted output here (you would see it if you read my post). Also, uptime means nothing, this is in house server as I said (you would know if you read my first post).

**Comment by UltimateByte** [comment]
If you had read the support link, you wouldn't be crying on github anymore, since it provides you with places to get support for this kind of issues. This link as well as the troubleshooting are helping everyone else, so I assume if it doesn't help you, it's because you either didn't read or didn't understand.

Your log output is 1) incomplete 2) not posted to pastebin or equivalent, 3) without any context provided. Like anyone should bother reading stuff like that...

I've been pointing you to docs that I wrote and that help solving most similar issues, because I've solved them a thousand times already, but still, you think you have tried everything and that I'm a jackass that won't help you, while on the other side, you're unable to post at the right place with the right info.

You sir, obviously have no sense of 1) how to get help by providing relevant info 2) following any kind of guidance 3) respect people that take time to answer your wrongly made posts made at the wrong place.
